website/docs/components/table/partials/code/component-api.md

```diff
@@ -41,9 +53,85 @@ The Table component itself is where most of the options will be applied. However
   </C.Property>
   <C.Property @name="...attributes">
     Supported for the `Hds::Table` component.
   </C.Property>
+  <C.Property @name="onSort" @type="function">
+    Callback function that is invoked when one of the sortable table headers is clicked (or has a keyboard interaction performed). The function receives the values of `sortBy` and `sortOrder` as arguments.
+  </C.Property>
+</Doc::ComponentApi>
+
+### Table::Tr
+
+This component is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). Instead, an interactive element should be placed _inside_ of the `Th`, `ThSort`, or `Td` elements.
+
+It can contain `Hds::Table::Th`, `Hds::Table::ThSort`, or `Hds::Table::Td` components.
+
+<Doc::ComponentApi as |C|>
+    <C.Property @name="...attributes">
+    This component supports the use of `...attributes`.
+  </C.Property>
+</Doc::ComponentApi>
+
+### Table::Th
+
+This component is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). Instead, an interactive element should be placed _inside_ of the `Th` element.
+
+If the `Th` component is passed as the first "cell" of a table body row, `scope="row"` is automatically applied for accessibility purposes.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="align" @type="enum" @values={{array "left" "center" "right" }} @default="left">
+    If set, determines the text alignment.
+  </C.Property>
+  <C.Property @name="scope" @type="string" @values={{array "col" "row" }} @default="col">
+    If used as the first item in a table body’s row, `scope` should be set to `row` for accessibility purposes. Note: you only need to manually set this if you’re creating a custom table using the child components, instead of the standard invocation.
+  </C.Property>
+  <C.Property @name="width" @type="string" @valueNote="Any valid CSS" @default="left">
+    If set, determines the column’s width.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports the use of `...attributes`.
+  </C.Property>
+</Doc::ComponentApi>
+
+### Table::ThSort
+
+This component is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). Instead, an interactive element should be placed _inside_ of the `ThSort` element.
+
+This is the component that supports the column sorting, and should be used instead of `Hds::Table::Th` if creating a custom implementation.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="isSorted" @type="boolean">
+    Set to true upon sort.
+  </C.Property>
+  <C.Property @name="sortOrder" @type="string" @values={{array "asc" "desc" }} @default="asc">
+    Use in conjunction with `sortBy`. If defined, indicates which direction the column should be pre-sorted in. If not defined, `asc` is applied by default.
+  </C.Property>
```

Comment:
please review this property declaration, is not correct
- "asc" is not the default for the `ThSort`([see here](https://github.com/hashicorp/design-system/blob/main/packages/components/addon/components/hds/table/th-sort.js#L37-L43))
- there's no relation with the `sortBy` (which is a `Table` argument): in this case `sortOrder` simply determines, if provided, the direction of the "sorting" icon and nothing more.

Reply:
We worked through this in a video call but documenting here for visibility. 
The columns are unsorted by default. If a pre-sort is desired, the author should add `@sortBy` with a value; the column would then be pre-sorted, ascending, by default. However, if the author desires to pre-sort the column in a descending order, they would add the `@sortOrder` as well.